synapse/models/infotech.py

```diff
@@ -934,6 +956,92 @@ def getModelDefs(self):
                         'doc': 'An array of ChildOf CWE Relationships.'
                     }),
                 )),
+
+                (('it:sec:metrics', {}, (
+
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The organization whose security program is being measured.'}),
+
+                    ('org:name', ('ou:name', {}), {
+                        'doc': 'The organization name. Used for entity resolution.'}),
+
+                    ('window', ('ival', {}), {
+                        'doc': 'The time window used to compute the metrics.'}),
+
+                    ('alert:meantime:triage', ('duration', {}), {
+                        'doc': 'The mean time to triage for alerts generated within the time window.'}),
+
+                    ('alert:count', ('int', {}), {
+                        'doc': 'The total number of alerts generated within the time window.'}),
+
+                    ('alert:falsepos', ('int', {}), {
+                        'doc': 'The number of alerts generated within the time window that were determined to be false positives.'}),
+
+                    ('assets:hosts', ('int', {}), {
+                        'doc': 'The total number of hosts within scope for the information security program.'}),
+
+                    ('assets:users', ('int', {}), {
+                        'doc': 'The total number of users within scope for the information security program.'}),
+
+                    ('assets:vulns:existing', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities being tracked at the beginning of the window.'}),
+
+                    ('assets:vulns:discovered', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities discovered during the window.'}),
+
+                    ('assets:vulns:mitigated', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities mitigated during the window.'}),
+
+                    ('assets:vulns:meantime:mitigate', ('duration', {}), {
+                        'doc': 'The mean time to mitigate for vulnerable assets mitigated during the time window.'}),
+
+                )),
+
+                ('it:sec:vuln:scan', {}, (
+
```

Comment:
it:sec:metrics assumes you have an accurate asset count :rofl: . Some scans (including routine/regular ones) may be used for asset discovery / identification. Or, in performing a true "vuln scan" you may identify additional assets you didn't know were there.

May be "out of scope" but tossing it out while we're spitballing.

Reply:
Def does not assume that so much as hope and optionally let you record if you do :)  Also didn't want to end up down the rabbit hole of "what constitutes an individual asset" :) Kicking this one down the road ;)